ui/components/Icon.tsx

```diff
@@ -167,6 +171,51 @@ function getIcon(i: IconType) {
     case IconType.PauseIcon:
       return PauseIcon;
 
+    case IconType.NotificationsBell:
+      return NotificationsBellIcon;
+
+    case IconType.SourcesIcon:
+      return () => (
+        <svg
+          width="24"
+          height="24"
+          viewBox="0 0 24 24"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          className="sources"
+        >
+          <path
+            d="M10.7 14.3L8.4 12L10.7 9.7L10 9L7 12L10 15L10.7 14.3ZM13.3 14.3L15.6 12L13.3 9.7L14 9L17 12L14 15L13.3 14.3V14.3Z"
+            fill="#1a1a1a"
+          />
+          <rect
+            x="4.5"
+            y="4.5"
+            width="15"
+            height="15"
+            rx="7.5"
+            stroke="#1A1A1A"
+          />
+        </svg>
+      );
+
+    case IconType.ImageAutomationIcon:
+      return () => (
+        <svg
+          width="24"
+          height="24"
+          viewBox="0 0 24 24"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
```

Comment:
FYI: the only downside of doing this inline (instead of a separate `.svg` file) is that the browser can't do fancy things to cache the image and our bundle size increases.

Not a big deal. I would keep as is.